frontend/testing/playwright/tests/dashboard/dashboard.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { expect } from '@playwright/test';
+import type { Page } from '@playwright/test';
+import { test } from '../../extenders/testExtend';
+import { DesignerApi } from '../../helpers/DesignerApi';
+import type { StorageState } from '../../types/StorageState';
+import { DashboardPage } from 'testing/playwright/pages/DashboardPage';
+import { OverviewPage } from 'testing/playwright/pages/OverviewPage';
+
+// This line must be there to ensure that the tests do not run in parallell, and
+// that the before all call is being executed before we start the tests
+test.describe.configure({ mode: 'serial' });
+
+// Before the tests starts, we need to create the dashboard app
+test.beforeAll(async ({ testAppName, request, storageState }) => {
+  // Create a new app
+  const designerApi = new DesignerApi({ app: testAppName });
+  const response = await designerApi.createApp(request, storageState as StorageState);
+  expect(response.ok()).toBeTruthy();
+
+  // Create another new app
+  const designerApi1 = new DesignerApi({ app: `${testAppName}2` });
+  const response1 = await designerApi1.createApp(request, storageState as StorageState);
+  expect(response1.ok()).toBeTruthy();
```

Comment:
should we extract this into a method with a descriptive name? Then we do not need to implement line 16-18 twice. 😊 
```suggestion
   const firstApp = createApp(testAppName);
   const secondApp = createApp(testAppName2);
   
   expect(firstApp.ok()).toBeTruthy();
   expect(secondApp.ok()).toBeTruthy();
```